tensorrt_llm/inputs/utils.py

```diff
@@ -465,10 +474,15 @@ def retrieve_all_sync(self) -> Optional[Dict[str, List[Any]]]:
 
         return {modality: items for modality, items in self._data.items()}
 
-    def add_data(self, media_type: str, data: Union[Coroutine, Any]):
+    def add_data(self,
+                 media_type: str,
+                 data: Union[Coroutine, Any],
+                 *,
+                 modality: Optional[str] = None):
+        modality = modality or media_type
```

Comment:
Why we need another `modality` signature in kwargs? Can we use `media_type` as modality?